frontend/src/components/view.js

```diff
@@ -10,9 +10,10 @@ import {
 
 import { HttpClient } from '../services/http';
 import { Settings } from '../settings';
-import { JenkinsJobView, JenkinsJobAnalysisView } from '../views';
+import { CompareRunsView, JenkinsJobView, JenkinsJobAnalysisView } from '../views';
 
 const VIEW_MAP = {
+  'compare-runs-view': CompareRunsView,
```

Comment:
Everywhere else your view is called `comparison-data` and then suddenly here you call it `compare-runs-view`. Please use the same name everywhere.

Reply:
`comparison-data` is for returning the data used in the Compare Runs view I created. `compare-runs-view` was for returning the view itself. Do these not need to be separate?

Reply:
Aaah, OK. I misunderstood - I thought there was only 1 thing in this, the comparison view.